libs/wingc/src/parser.rs

```diff
@@ -681,8 +681,18 @@ impl<'s> Parser<'s> {
 			return self.add_error(format!("Syntax error"), &nested_node);
 		}
 		if let Some(property) = nested_node.child_by_field_name("property") {
+			let object_expr = self.get_child_field(nested_node, "object")?;
+			let object_expr = if object_expr.kind() == "json_container_type" {
+				Expr {
+					kind: ExprKind::Reference(Reference::Identifier(self.node_symbol(&object_expr)?)),
+					span: self.node_span(&object_expr),
+					evaluated_type: RefCell::new(None),
+				}
+			} else {
+				self.build_expression(&nested_node.child_by_field_name("object").unwrap())?
+			};
 			Ok(Reference::InstanceMember {
```

Comment:
See comment above about creating a `Reference::TypeMember` in case of an `xxx_container_type`.